joomla/joomla-cms
# [com_messages] Implement searchtools

Pull Request for Improvement.
#### Summary of Changes

Like done for so many other views across Joomla backend, this PR adds the search tools to com_messages messages view.
Also improved mobile view, added a "All" filter in the message state and some minor adjustements.
###### Before Patch

![image](https://cloud.githubusercontent.com/assets/9630530/14467093/a3d2a048-00d1-11e6-8163-522a66e312d7.png)
###### After Patch

![image](https://cloud.githubusercontent.com/assets/9630530/14467065/7dc72432-00d1-11e6-9ab2-a6a7a4a03a73.png)
#### Testing Instructions
1. Apply patch
2. Go to "Components" -> "Messaging"
3. Send some Messages across users (you will need to create another user for that)
4. Check if the searchtools, filters and ordering are working properly.
